docker-compose.test.yml

```diff
@@ -4,13 +4,23 @@ services:
     image: libero/continuum-adaptor:${IMAGE_TAG:-local}
     container_name: test_continuum_adaptor
     ports:
-      - "${AUTHENTICATION_PORT:-3001}:3001"
+      - "3001:3001"
     environment: 
-      CONFIG_PATH: /etc/reviewer/config.json
-    volumes:
-      - ./tests/config/continuum-adaptor.json:/etc/reviewer/config.json
+      PORT: 3001
+      RABBITMQ_URL: rabbitmq
+      LOGIN_URL: http://localhost:9000/submit
+      LOGIN_RETURN_URL: http://localhost:3003/redirect_location_for_intergration_test
+      AUTHENTICATION_JWT_SECRET: super_secret_jam
+      CONTINUUM_JWT_SECRET: some_secret_from_journal
+      CONTINUUM_API_URL: http://reviewer-mocks:3003
+      DATABASE_NAME: postgres
+      DATABASE_USER: postgres
+      DATABASE_PASSWORD: postgres
+      DATABASE_HOST: postgres
+      DATABASE_PORT: 5432
+      ELIFE_API_GATEWAY_SECRET: ${ELIFE_API_GATEWAY_SECRET}
     healthcheck:
-      test: 'echo -e "GET /health\n\n" | nc localhost ${AUTHENTICATION_PORT:-3001}'
+      test: 'echo -e "GET /health\n\n" | nc localhost 3001'
```

Comment:
Are we sure the env var gets expanded in the container and not on the host? (does it matter?)